components/calendar/src/iso.rs

```diff
@@ -136,13 +161,13 @@ impl Calendar for Iso {
         // https://en.wikipedia.org/wiki/Determination_of_the_day_of_the_week#Corresponding_months
         let month_offset = if Self::is_leap_year(date.year) {
             match date.month.0 {
-                1 | 4 | 7 => 0,
-                10 => 1,
-                5 => 2,
-                2 | 8 => 3,
-                3 | 11 => 4,
-                6 => 5,
-                9 | 12 => 6,
+                10 => 0,
```

Comment:
Note to self: This is shifting from 0-indexed to 1-indexed months

Reply:
It's shifting form 0-indexed to 1-indexed _offsets_, it's the rhs that's changing